src/util.js

```diff
@@ -3,7 +3,7 @@ define([
     'underscore',
     'jsdiff',
     'vellum/markdown',
-    'vellum/xpath',
+    'vellum/escapedHashtags',
```

Comment:
This didn't actually change. `xpath` just wasn't used, since the form contains its own xpath parser. it is now using a utility from `escapedHashtags`